store/store.go

```diff
@@ -487,3 +487,7 @@ func (s *store) pruneBlock(blockHeight uint32) (bool, error) {
 
 	return true, nil
 }
+
+func (s *store) RetentionBlocks() uint32 {
```

Comment:
do we need it?

Reply:
Yes, we can't access it from state otherwise.